### ISSUE TITLE
Some words no longer have meaning displayed

### ISSUE THREAD

**Issue by krofna** [Gnurou/tagainijisho#233]
I think it happened after dictionary update. I'm not sure.
![image](https://user-images.githubusercontent.com/1102695/87875504-e5dc7680-c9d1-11ea-9b56-572e2993872a.png)

**Comment by Gnurou** [comment]
Mmm, that's not looking good. Is this a source build from git? Released versions don't see their dictionary updated.

**Comment by krofna** [comment]
Yes it's a source build.

**Comment by Gnurou** [comment]
I have tried a fresh build, including re-downloading JMdict, and things are working as expected. Could you try removing the `3rdparty/JMdict` file and then try to build again starting from `cmake`? Maybe we got a defective JMdict for some reason.

**Comment by krofna** [comment]
I did it. About window says `JMDict version 2020-07-19` but still the same words (e.g. 御主) have the same issue. I also ran it as different user (without my database) and the same issue is present.

**Comment by Gnurou** [comment]
I have the same JMdict version (md5sum 13430a94b5264bfb0d759e2b201ae9d4 for the record), so there is something else at work it seems.

What is weird is that on your screenshot the parts of speech are not even appearing. Could you check in the preferences what the preferred GUI language and preferred dictionary language are, and maybe mess with them a bit? Checking "Always show English meanings" may help as well.

**Comment by krofna** [comment]
My JMdict file has the same hash.

I tried messing with the options but nothing works. It would be strange if it did since it only affects some words. 
![image](https://user-images.githubusercontent.com/1102695/87940190-7fbb2680-ca99-11ea-896a-51a3980fb2ec.png)

**Comment by Kienyew** [comment]
Another random user here, I can confirm this problem also occurred on me before although it doesn't affect my using experience too much. On the word 「御主」, my situation is same as @krofna 's.
![image](https://user-images.githubusercontent.com/31496021/87941245-6af88600-cacd-11ea-89e6-ad6e6661f550.png)

**Comment by Gnurou** [comment]
Ah, so it's not *all* words, only some that lose their definitions.

Oh, actually... I noticed that 御主 has an 'archaic' POS. There is an option to filter these entries in Preferences -> Vocabulary entries -> Definitions filtering. I have added "Archaism" to the filtered definitions and am now getting the same behavior as you. You may want to remove it from there if you want such entries to appear.

Note that this is still a bug, because the idea of these filters is to remove the entry entirely, not just the definition.

Maybe I should just keep going on my simplification marathon and remove these filters, just like I am removing fonts customization. They seem to get in the way more than they are helping.

**Comment by Gnurou** [comment]
Hey @krofna, just would like to confirm whether the little trick in my last comment fixed the issue for you - could you confirm?

**Comment by krofna** [comment]
Yeah, now they're displayed again. I'd forgotten about the definition filtering preferences.

**Comment by Gnurou** [comment]
Great, thanks for confirming. I'll keep this issue opened until the issue is properly addressed one way or the other.
